internal/database/migrations/migrations.go

```diff
@@ -51,6 +56,13 @@ func parseJson(filename string, obj interface{}) error {
 	return nil
 }
 
+func ChooseRandomTag() string {
+	tagChoice := []string{"EDUCATION", "REFUGEE", "IMMIGRATION"}
+	randomIndex := rand.Intn(len(tagChoice))
```

Comment:
oh ya, do we have to seed before usage?

Reply:
yeah or add story through form which has tags